scripts/test-runner-lambdas.py

```
@@ -0,0 +1,590 @@
+        return module
+
+
+def load_lambda(source, ec2, ssm, lambda_client=None, github=None, env=None, now=NOW):
+    """exec the Lambda source with its AWS and GitHub edges replaced."""
+    clients = {"ec2": ec2, "ssm": ssm, "lambda": lambda_client}
+    fake_boto3 = types.ModuleType("boto3")
+    fake_boto3.client = lambda service, **kw: clients[service]
+    sys.modules["boto3"] = fake_boto3
+
+    namespace = {"__name__": "lambda_function"}
+    exec(compile(source, "<lambda_function.py>", "exec"), namespace)
```

Ruff (0.16.1)

[error] Use of `exec` detected

(S102)